pkg/storagev2/storage.go

```diff
@@ -65,6 +65,8 @@ var ErrInvalidQuery = errors.New("storage: invalid query")
 
 var ErrNotFound = errors.New("storage: not found")
 
+var ErrReferenceLength = errors.New("invalid reference length")
+
```

Comment:
would it make sense to follow through convention when declaring multiple variables to use singe var block just like constants are declared (few lines above)?

is `ErrReferenceLength` missing text prefix `storage: `, following other errors?